extra_data/tests/test_components.py

```diff
@@ -104,6 +104,14 @@ def test_get_array_pulse_indexes_reduced_data(mock_reduced_spb_proc_run):
     assert np.isin(arr.coords['pulse'], [1, 7, 15, 23]).all()
 
 
+def test_get_array_as_xarray(mock_fxe_raw_run):
```

Comment:
Naming again - it returns an xarray object either way, so it's not quite right to call this test `_as_xarray`.